stdlib/Serialization/src/Serialization.jl

```diff
@@ -39,7 +39,7 @@ const TAGS = Any[
     Float16, Float32, Float64, Char, DataType, Union, UnionAll, Core.TypeName, Tuple,
     Array, Expr, LineNumberNode, :__LabelNode__, GotoNode, QuoteNode, CodeInfo, TypeVar,
     Core.Box, Core.MethodInstance, Module, Task, String, SimpleVector, Method,
-    GlobalRef, SlotNumber, TypedSlot, NewvarNode, SSAValue,
```

Comment:
Hmm, it looks like we need to replace this `TypedSlot` with something defined in `Core`?

Reply:
We could just convert to a `SlotNumber` on load from this old tag id (ignore the typ), though I don't think anyone would be using this

Reply:
I think you need to at least keep a placeholder here though to keep the numbering constant

Reply:
Can `Const` for example be a placeholder here? Or is something other better?